packages/edit-site/src/components/sidebar-navigation-screen/style.scss

```
@@ -127,9 +127,14 @@
 	bottom: 0;
 	background-color: $gray-900;
 	gap: 0;
-	padding: $grid-unit-20 0;
+	padding: $grid-unit-10 $grid-unit-20;
 	margin: $grid-unit-20 0 0;
 	border-top: 1px solid $gray-800;
+
+	.components-item-group {
```

```suggestion
	.edit-site-sidebar-navigation-screen-details-footer {
```

Let's avoid using `.components`-prefixed CSS classes whenever possible.